.github/workflows/master.yml

```diff
@@ -177,6 +178,12 @@ jobs:
         mkdir -p ~/.m2
         ./dev/run-tests --parallelism 2 --modules "$MODULES_TO_TEST" --included-tags "$INCLUDED_TAGS" --excluded-tags "$EXCLUDED_TAGS"
         rm -rf ~/.m2/repository/org/apache/spark
+    - name: "Publish test report: ${{ matrix.modules }}"
+      if: always()
+      uses: scacap/action-surefire-report@v1
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think I should file an infra ticket and handle this github token. I will proceed separately.

Reply:
It's in progress at INFRA-20628.

Reply:
What the issue here?

Reply:
I'll leave a comment below.